internal/engine/engine.go

```diff
@@ -217,20 +217,22 @@ func (c *craneEngine) ExecuteChecks(ctx context.Context) error {
 	// execute checks
 	logger.V(log.DBG).Info("executing checks")
 	for _, executedCheck := range c.checks {
+		chkLogger := logger.WithValues("check", executedCheck.Name())
```

Comment:
Since this is inside a loop, I think this can be a two line change. Just shadow ctx and logger. It will not effect the outer values, and will get thrown away at the end of the iteration. I don't see any instances where we use the old values.

Reply:
Yep, initially thought it was less clear. I'll swap it out.